### PR TITLE
Applies misc DB updates to project funding

### PR DESCRIPTION
**Pull request by johnclary**
## Associated issues
- https://github.com/cityofaustin/atd-data-tech/issues/13113

## Testing
**URL to test:** Local

**Steps to test:**
1. start the hasura cluster and editor
2. visit any project's "Funding" tab.
3. Add a new funding source.
 - Verify that `CapMetro ILA 2018` is listed in the **Source** dropdown menu
 - Verify these options in the **Program** dropdown menu
   - `Large CIP`
   - `Sidewalk Fee in Lieu`
   - `Sidewalk Rehab`
   - `Street Impact Fee`
   - `Street Rehabilitation`
   - `Traffic Mitigation Fees`
   - `Transit Enhancement Program`

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable

### CODE REVIEW

File: moped-database/migrations/1690917480404_proj_funding_updates/up.sql [hunk 1]
```diff
@@ -0,0 +1,26 @@
+SELECT
+    setval('moped_fund_source_cat_funding_source_category_id_seq', (
+            SELECT
+                MAX(funding_program_id)
+                FROM moped_fund_programs));
```
Comment (johnclary) [inline moped-database/migrations/1690917480404_proj_funding_updates/up.sql:5]: at least locally, this sequence was out of whack when i tried to insert new records. this resets it.

**Comment by johnclary** [comment]
👋 this is not critical for the upcoming release: i am unblocked on the access migration.

File: moped-database/migrations/1690917480404_proj_funding_updates/up.sql (hunk 1 above)
Comment (frankhereford) [reply moped-database/migrations/1690917480404_proj_funding_updates/up.sql:5]: Thanks for sorting this out. I couldn't recall if sequences store the next value or the current value, so I threw together this tiny test case, and it this is perfect. 👍 

```sql
CREATE TABLE frank_test (
    id SERIAL PRIMARY KEY,
    data INTEGER
);

insert into frank_test (data) values (1);
select * from frank_test;

SELECT currval('frank_test_id_seq');

insert into frank_test (data) values (1);
select * from frank_test;

SELECT currval('frank_test_id_seq');
```

**Review by frankhereford — APPROVED** [review]
I spun this up and tested it with your instructions. Works great. The code looks great too - 🚢🚢🚢!

**Review by mddilley — APPROVED** [review]
Tested locally, steps look great, code looks great. All great! 🚀

**Comment by johnclary** [comment]
Thanks y'all! Appreciate that extra test @frankhereford!

**Pull request merged**